src/pages/api/bugs/[id].js

```diff
@@ -0,0 +1,76 @@
+import sequelize from '../../../../server/config/connection.js';
+import { Bug } from '../../../../server/models';
+
+export default handler;
+
+function handler(req, res) {
+    switch (req.method) {
+        case 'GET':
+            return getBugById(req.params.id);
+            
+        case 'PUT':
+            return updateBug(req.params.id, req.body.status, req.body.priority);
+            
+        case 'DELETE':
+            return deleteBug(req.params.id);
+            
+        default:
+            return res.status(405).end(`Method ${req.method} Not Allowed`);
+
+    };
+
+    async function getBugById(id) {
+        try {
+            const bug = await Bug.findByPk(id)
+            res.status(200).json({ bug });
+
+        } catch (error) {
+            return res.status(400).json({ 'message': 'Could not connect to database' });
+        }
+
+    };
+
+    async function updateBug() {
+
+
+        //if the params title already exists for another bug will throw an error
+        if (req.params.title !== Bug.title || Bug.findByPk(req.body.title))
```

Comment:
It's ok if there are duplicate titles. also you want to `throw new Error()` and wrap the `Error` around the message